pkg/machines/helpers.es6

```diff
@@ -192,7 +192,7 @@ export function fileDownload ({ data, fileName = 'myFile.dat', mimeType = 'appli
     window.setTimeout(() => { // give phantomJS time ...
         logDebug('removing temporary A.HREF for filedownload');
         document.body.removeChild(a);
-    }, 500);
+    }, 5000);
```

Comment:
The temporary link might become visible within this timewindow.
If so, we can either hide it (not sure whether it breaks phantom test or not) or just set link's color to the backgound's color